Update installation.md (#2608)
MithrilJS/mithril.js: docs/installation.md

7. open `index.html` in a browser

Optionally, you can define mithril as a webpack plugin, the benefit is that mithril will be included globally and you wont have have to `import m from "mithril"`:
```json
plugins: [
    new webpack.ProvidePlugin({m: 'mithril'}),
    // ...

>     ]

Reviewer: Please keep the indentation level for this consistent with everything else.

```suggestion
]
```

```

> Then you could remove the import line from step 4 (dont forget to restart webpack if you ran it with `--watch`) and it will work just the same

Reviewer: Please add a newline after this for consistency with everything else.

Author: Done, github doesnt show it in the main Conversations because it takes it as part of the following block for some reason but if you check [today's commit](https://github.com/MithrilJS/mithril.js/pull/2608/commits/740620c1fe451bd113a6e8056316eb171041391c), its there

7. open `index.html` in a browser

> Optionally, you can define Mithril as a Webpack plugin, the benefit is that Mithril will be included globally and you will not have have to `import m from "mithril"`:

Reviewer: ```suggestion
Optionally, you can include Mithril as a global variable using Webpack's provide plugin, to avoid including `import m from "mithril"` across a large number of files:
```

> ```json

Reviewer: So it'll highlight properly

````suggestion
```js
````

plugins: [

>     new webpack.ProvidePlugin({m: 'mithril'}),

Reviewer: For consistency

```suggestion
    new webpack.ProvidePlugin({m: "mithril"}),
```

    // ...
]
```

> Then you could remove the import line from step 4 (don't forget to restart Webpack if you ran it with `--watch`) and it will work just the same.

Reviewer: ```suggestion
Then, you could remove the import line from step 4 (don't forget to restart Webpack if you ran it with `--watch`), and it will work just the same.
```
